translations/ru/tutorial/index.md

```diff
@@ -0,0 +1,39 @@
+---
+
+Добро пожаловать в Gatsby! Мы рады, что ты здесь. Цель этого руководства ― помочь настроить и развернуть твой первый сайт с использованием Gatsby. В этом руководстве мы затронем некоторые темы веб-разработки и рассмотрим основную структуру сайта Gatsby.
+
+> Данное руководство предназначено для людей, не имеющих (пока что) большого опыта веб-разработки. Если тебя не интересует базовая подготовка, смело переходи на страницу [быстрого старта](/docs/quick-start/).
+
+## Основы Gatsby.js
+
+0.  [Настройка среды разработки](/tutorial/part-zero/): Познакомимся с основными технологиями Gatsby и настроим среду разработки.
+1.  [Строительные блоки Gatsby](/tutorial/part-one/): Рассмотрим процессы создания проектов, их разработки и развертывания.
+1.  [Вступление в стилизацию Gatsby](/tutorial/part-two/): Изучим библиотеки, такие как Typography.js и CSS Modules в контексте Gatsby приложения.
+1.  [Создание шаблонов и компонентов](/tutorial/part-three/): Создадим шаблонные компоненты, которые могут использоваться на нескольких страницах, такие как хедер и футер.
+
+## Расширенные руководства
+
+В разделах ниже ты узнаешь как получать данные практически из любого источника с помощью GraphQL.
+
+4. [Получение данных для блога](/tutorial/part-four): Создадим блог и используя GraphQL, вставим заголовок сайта в блог.
+5. [Встроенные плагины и рендеринг запрашиваемых данных](/tutorial/part-five): С помощью плагинов встроим markdown-статьи на сайт и создадим страницу со списком всех постов блога.
+6. [Плагины-трансформеры](/tutorial/part-six): Поработаем с плагинами-трансформерами, чтобы преобразовать markdown в читабельный вид, с дальнейшим отображением в блоге.
+7. [Программное создание страниц из данных](/tutorial/part-seven): Познакомимся с API для программного создания страниц из списка публикаций.
+8. [Подготовка сайта к работе](/tutorial/part-eight): Проведем аудит сайта на предмет его производительности, SEO, a11y и многому другому.
+
+## Руководства по работе с темами
+
+Темы Gatsby абстрагируют конфигурацию по умолчанию (общий функционал, получение данных, дизайн) с сайта в устанавливаемый пакет.
+
+Это означает, что конфигурация и функциональность не находятся непосредственно в проекте, а скорее являются версионированными, управляемыми и устанавливаемыми зависимостями. Темы можно легко обновлять, объединять несколько в одну и даже менять одну совместимую тему на другую.
+
+Узнать больше об использовании и создании тем можно с помощью [руководств по темам Gatsby](/tutorial/theme-tutorials/).
+
+## Дополнительные руководства
+
+Есть много вариантов использования Gatsby и не все из них затрагиваются в основных руководствах, чтобы сосредоточить тебя во время обучения. Раздел "Дополнительные руководства Gatsby" представляет собой набор ресурсов о более широком спектре задач Gatsby, например плагины для изображений и работа с CMS.
+
+Изучай больше с ["дополнительными руководствами Gatsby"](/tutorial/additional-tutorials/).
```

Comment:
```suggestion
Изучай больше с ["дополнительными руководствами Gatsby"](/tutorial/additional-tutorials/).

```

This should have a line `40` in blank like this to match the original file 40 lines :rocket:

Reply:
Strange, because locally a had 40 lines in both files:
![image](https://user-images.githubusercontent.com/28801003/65768565-f61d1f80-e139-11e9-9aa4-f0630d9fe902.png)